dir.props

```diff
@@ -559,6 +580,9 @@
     <PackagesBasePath Condition="'$(PackagesBasePath)'==''">$(BinDir)$(OSPlatformConfig)</PackagesBasePath>
     <PackageOutputPath Condition="'$(PackageOutputPath)'==''">$(PackageOutputRoot)$(ConfigurationGroup)/</PackageOutputPath>
     <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)symbols/</SymbolPackageOutputPath>
+  
+    <!-- When testing against packages, we want to run against just the localpackages cache -->
+    <PackagesDir Condition="'$(BuildTestsAgainstPackages)'=='true'">$(PackageInstallPath)</PackagesDir>
```

Comment:
Have you verified that code coverage runs work with these changes?  I would expect them to fail due to not finding the TestHost if this is indeed an issue.